all-spark-notebook/Dockerfile

```diff
@@ -20,11 +20,11 @@ RUN apt-get update && \
 USER $NB_USER
 
 # R packages
-RUN conda config --add channels r && \
+RUN conda config --add channels rdonnellyr && \
```

Comment:
On the main R channel I was not able to find r-base 3.3.  I was getting the error `Error: Package missing in current linux-64 channels: `  Based on the [documentation](https://docs.continuum.io/anaconda/r-language-pkg-docs) and [packages] (http://repo.continuum.io/pkgs/r/linux-64/) it doesn't look to be there now.

Just found this issue that explains the situation https://github.com/conda/conda-recipes/issues/683.  The test package is in rdonnellyr and once that is verified (soon) will be moved to the main R channel.  Based off that we can hold off merging this PR for a few more days till it is in the main R channel.